src/modules/account/store/reducer.js

```diff
@@ -1,19 +1,55 @@
 import { combineReducers } from 'redux';
+import { persistReducer } from 'redux-persist';
+import { storage } from '../../../../packages/common/store/index';
 import actionTypes from './actionTypes';
 
 /**
  *
  * @param {Object} state
  * @param {type: String, accountSchema: Object} action
  */
```

Comment:
Please complete this documentation.

Reply:
Please do not resolve the comment without fixing it, or leaving a comment. Otherwise why do we bother with reviewing the PRs?